pennylane/wires.py

```diff
@@ -81,37 +68,33 @@ class Wires(Sequence):
     """
 
     def __init__(self, wires):
-        self._labels = _process(wires)
-
-    def __getitem__(self, idx):
-        """Method to support indexing. Returns a Wires object representing a single wire."""
-        return Wires(self.labels[idx])
-
-    def __len__(self):
-        """Method to support ``len()``."""
-        return len(self.labels)
+        """ Extend the OrderedSet.__init__ by allowing for structured inputs """
```

Comment:
No need to have a docstring on an inherited method (unless the logic/signature differs)
```suggestion
```